yarn-project/bb-prover/src/bb/cli.ts

```diff
@@ -88,5 +88,40 @@ export function getProgram(log: LogFn): Command {
         log,
       );
     });
+
+  program
+    .command('write-contract')
+    .description('Generates the verification contract for the specified circuit')
+    .requiredOption(
+      '-w, --working-directory <string>',
+      'A directory to use for storing input/output files',
+      BB_WORKING_DIRECTORY,
+    )
+    .requiredOption('-b, --bb-path <string>', 'The path to the BB binary', BB_BINARY_PATH)
+    .requiredOption('-c, --circuit <string>', 'The name of a protocol circuit')
+    .requiredOption('-cn --contract-name <string>', 'The name of the contract to generate', 'contract.sol')
```

Comment:
Super nit: two character short flags are weird to me. Maybe just `-n` for contract name?